assets/l10n/app_en.arb

```diff
@@ -1419,5 +1419,9 @@
   "zulipAppTitle": "Zulip",
   "@zulipAppTitle": {
     "description": "The name of Zulip. This should be either 'Zulip' or a transliteration."
-  }
+  },
+   "loading": "Loading…",
+    "@loading": {
+      "description": "Label shown for loading indicators"
```

Comment:
nit: "Label shown for loading indicators" is misleading; this isn't shown in the app's visual UI. Rather, it's information presented to screen-reader software, to help create an interface for users with vision loss etc.

How about: "Semantic label for a loading indicator."